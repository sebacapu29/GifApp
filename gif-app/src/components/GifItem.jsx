import Card from 'react-bootstrap/Card';
import PropTypes from 'prop-types'

export const GifItem = ({title, url, id}) => {
  return (
    <Card>
          <Card.Img variant="top" src={url} />
          <Card.Body>
             <Card.Title>{title}</Card.Title>
           </Card.Body>
        </Card>
  );
}

GifItem.prototype = {
  title: PropTypes.string.isRequired,
  url: PropTypes.string.isRequired
}