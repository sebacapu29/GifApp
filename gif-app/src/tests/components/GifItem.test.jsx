import {render} from '@testing-library/react'
import {GifItem} from '../../components/GifItem';

describe('Pruebas en <GifItem />', () => {

    const title = 'Sitama';
    const url = 'https://web-gifs.com/saitama.jpg';

  test('debe de hacer match con el snapshot', () => {
    const { container } = render(<GifItem title={title} url ={url}/>)
    expect(container).toMatchSnapshot();
})
  
})
