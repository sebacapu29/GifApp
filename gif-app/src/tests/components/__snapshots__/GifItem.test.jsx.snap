// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en <GifItem /> debe de hacer match con el snapshot 1`] = `
<div>
  <div
    class="card"
  >
    <img
      class="card-img-top"
      src="https://web-gifs.com/saitama.jpg"
    />
    <div
      class="card-body"
    >
      <div
        class="card-title h5"
      >
        Sitama
      </div>
    </div>
  </div>
</div>
`;
